specifyweb/frontend/js_src/lib/components/Leaflet/localityRecordDataExtractor.ts

```diff
@@ -136,7 +136,8 @@ async function recursiveResourceResolve(
       return [];
     const tableRanks = strictGetTreeDefinitionItems(
       treeTableName as 'Geography',
-      false
+      false,
+      undefined
```

Comment:
passing undefined here seems needless as that would be the default behavior when attribute is not provided

is your intention to not-apply the default value? default value still applies if the passed attribute is undefined

Reply:
yes, this might be one of the only few good reasons to use `null`
i.e `undefined` could be though of as `I provide no value - feel free to use the default`
where as `null` is `I explicitly tell you that there is no value!`
but your idea of using `'all'` is much better as it's more readable than seeing `null`